src/macfc-security-hub-sync.ts

```diff
@@ -186,6 +374,7 @@ export class SecurityHubJiraSync {
     return updatesForReturn
   }
   makeResourceList(resources: Resource[] | undefined) {
+    console.log('Resources: ', resources?.length)
```

Comment:
Is this console.log necessary when it is not in debug mode?

Reply:
consolidated